contracts/Client.sol

```diff
@@ -0,0 +1,222 @@
+pragma solidity ^0.4.10;
+
+contract Client {
+  struct PersonalInfo {
+    bytes32 name;
+    bytes32 homeAddress;
+    bytes32 birthday;
+    uint256 age; 
+    uint8 gender; // gender 0 = female, 1 = male
+    uint8 education; // 0 = none / 1 = ma / 2 = hs / 3 = college / 4 = masters
+    uint8 householdSize;
+    uint8 dependents;
+    bool married;
+    uint256 monthlyHouseholdIncome;
+    uint8 numOfBusinesses;
+    uint8 numOfEmployees; 
+    uint256 phoneNumber; // 0 for doesn't have one
+  }
+  PersonalInfo public clientInfo;
+  address public clientWallet;
+
+  mapping ( uint256 => Job ) public jobHistory;
+  uint256 public numberOfJobs;
+  struct Job {
+    bytes32 title;
+    bytes32 employer;
+    bytes32 workAddress;
+    bytes32 startDate;
+    bytes32 endDate;
+    uint256 monthlySalary;
+  }
+
+  function addJob(
+    bytes32 _title,
+    bytes32 _employer,
+    bytes32 _workAddress,
+    bytes32 _startDate,
+    bytes32 _endDate,
+    uint256 _monthlySalary
+  ) {
+    numberOfJobs = numberOfJobs + 1;
+    Job memory newJob = Job(_title, _employer, _workAddress, _startDate, _endDate, _monthlySalary);
+    jobHistory[numberOfJobs] = newJob;
+  }
```

Comment:
Seems like permissions should be here as well (maybe only client can add a job?)

Reply:
word, good thinking gonna restrict it to the mfi adding a job - since the clients really only have read access to this